client/src/lit/curriculum/scrim-inline.js

```diff
@@ -71,6 +79,22 @@ export class ScrimInline extends LitElement {
     }
   }
 
+  /**
+   * @param {MouseEvent} event
+   */
+  _handleVote({ target }) {
+    if (target instanceof HTMLElement) {
+      const vote = target.dataset.vote;
+      const label = vote === "better" ? "1" : vote === "worse" ? "-1" : "0";
+      this._voted = true;
+
+      gleanClick("thumbs", {
+        type: "scrim-challenges",
+        label: label,
+      });
+    }
```

Comment:
You need to use [`_gleanClick()`](https://github.com/search?q=repo%3Amdn%2Fyari%20%2F_gleanClick%2F&type=code) provided by the [`GleanMixin`](https://github.com/search?q=repo%3Amdn%2Fyari%20%2FGleanMixin%2F&type=code).